README.md

```diff
@@ -43,65 +43,297 @@
+    <details>
+    <summary>See config</summary>
+    <br>
+
+    ```toml
+    # Schemas needed for our coop notice board application
+    allow_schema_ids = [
+        "notice_board_0020c3accb0b0c8822ecc0309190e23de5f7f6c82f660ce08023a1d74e055a3d7c4d",
+        "notice_board_post_0020aaabb3edecb2e8b491b0c0cb6d7d175e4db0e9da6003b93de354feb9c52891d0",
+    ]
+
+    # Addresses of already known nodes we can connect directly to
+    direct_node_addresses = [
+        "192.0.2.78:2022",
+        "198.51.100.22:2022",
+        "192.0.2.211:2022",
+        "203.0.114.123:2022",
+    ]
+
+    # Peer ids of allowed peers, these will be the expected identities for the nodes we are connecting
+    # directly to
+    allowed_peer_ids = [
+        "12D3KooWP1ahRHeNp6s1M9qDJD2oyqRsYFeKLYjcjmFxrq6KM8xd",
+        "12D3KooWPC9zdWXQ3aCEcxvuct9KUWU5tPsUT6KFo29Wf8jWRW24",
+        "12D3KooWDNNSdY8vxYKYZBGdfDTg1ZafxEVuEmh49jtF8rUeMkq2",
+        "12D3KooWMKiBvAxynLn7KmqbWdEzA8yq3of6yoLZF1cpmb4Z9fHf",
+    ]
+    ```
+    </details>
+
+We're excited to hear about your ideas! Join our [official chat](https://wald.liebechaos.org/) and reach out.
+
+### GraphQL query API
+
+As an application developer the interface to `aquadoggo` you will use the most is the GraphQL query API. For whichever schema your node supports a custom query api is generated, you use this to fetch data into your app. Results from a collection query can be paginated, filtered.
+
+Fetch one "mushroom" by it's id, returning values for only the selected fields:
+```graphql
+{
+  mushroom: mushroom_0020c3accb0b0c8822ecc0309190e23de5f7f6c82f660ce08023a1d74e055a3d7c4d(
+    id: "0020aaabb3edecb2e8b491b0c0cb6d7d175e4db0e9da6003b93de354feb9c52891d0"
+  ) {
+    fields {
+      description
+      edible
+      latin
+      title
+    }
+  }
+}
+```
+
+<details>
+<summary>Example query response</summary>
+<br>
+
+```json
+{
+  "mushroom": {
+    "description": "Its scientific name rhacodes comes from the Greek word rhakos, which means a piece of cloth. It does often have a soft, ragged fabric-like appearance.",
+    "edible": true,
+    "latin": "Chlorophyllum rhacodes",
+    "title": "Shaggy parasol"
+  }
+}
+```
+</details>
+
+A collection query for "events" which includes ordering and filtering as well as selecting some meta fields. Here only events between the specified dates and with a title containing the string 'funtastic' will be returned, they will be arranged in ascending chronological order:
+```graphql
+{
+  events: all_events_0020aaabb3edecb2e8b491b0c0cb6d7d175e4db0e9da6003b93de354feb9c52891d0(
+    first: 20
+    orderBy: "happening_at"
+    orderDirection: ASC
+    filter: {
+      title: { contains: "funtastic" }
+      happening_at: { gte: 2023-08-31, lte: 2023-09-14 }
```

Comment:
Think the values need to be an array

Reply:
Which values?

Reply:
The `happening_at` values need to be unix timestamps